src/XMLSecurityDSig.php

```diff
@@ -99,54 +120,91 @@ public function __construct($prefix='ds')
         $this->sigNode = $sigdoc->documentElement;
     }
 
-    private function resetXPathObj()
```

Comment:
PHPStorm has a check box in the reformating dialog for only touch VCS changed lines. Be sure of to have it enabled.